setup.py

```diff
@@ -5,30 +5,29 @@
 from setuptools import setup
 import versioneer
 
-# NOTE: These are tested in `continuous_integration/travis/test_imports.sh` If
+# NOTE: These are tested in `continuous_integration/test_imports.sh` If
 # you modify these, make sure to change the corresponding line there.
 extras_require = {
-    "array": ["numpy >= 1.15.1", "toolz >= 0.8.2"],
-    "bag": [
-        "cloudpickle >= 0.2.2",
-        "fsspec >= 0.6.0",
-        "toolz >= 0.8.2",
-        "partd >= 0.3.10",
+    "array": [
+        "numpy >= 1.15.1",
     ],
+    "bag": [],
     "dataframe": [
         "numpy >= 1.15.1",
         "pandas >= 0.25.0",
-        "toolz >= 0.8.2",
-        "partd >= 0.3.10",
-        "fsspec >= 0.6.0",
     ],
     "distributed": ["distributed >= 2021.03.0"],
     "diagnostics": ["bokeh >= 1.0.0, != 2.0.0"],
-    "delayed": ["cloudpickle >= 0.2.2", "toolz >= 0.8.2"],
```

Comment:
Could we keep this for backwards compat too?

Reply:
:facepalm: yes.